lib/dom.js

```diff
@@ -12,7 +12,20 @@
 
 const jsdom = require('jsdom').jsdom;
 
-var addChildren = function(/*DOM*/ elem, /*function*/ filter, /*JsonML*/ jml) {
+/**
+ * A JsonML node represented as an Array or string value
+ * @typedef {Array/string} JsonML
+ */
+
+/**
+ * Add children to a JsonML node from a given DOM element
+ *
+ * @param {Element} elem - DOM Element
```

Comment:
`Node` and `Element` are used almost interchangeably throughout this library but they are not not one and the same. Whether nor or later it would be good to clearly resolve this distinction to be sure all documentation and variable naming is accurate.